<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDInfosConnection</class>
 <widget class="QDialog" name="BDInfosConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Nom de BD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNomBD"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>31</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Serveur:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditServeur"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>21</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Utilisateur:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUser"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mot de passe:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>49</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BDInfosConnection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BDInfosConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
